examples/direct_fidelity_estimation_test.py

```diff
@@ -51,13 +49,12 @@ def test_direct_fidelity_estimation_with_noise():
     noise = cirq.ConstantQubitNoiseModel(cirq.depolarize(0.1))
     noisy_simulator = cirq.DensityMatrixSimulator(noise=noise)
 
-    estimated_fidelity = direct_fidelity_estimation.direct_fidelity_estimation(
-        circuit,
-        qubits,
-        noisy_simulator,
-        n_trials=10,
-        n_clifford_trials=3,
-        samples_per_term=10)
+    estimated_fidelity, _ = dfe.direct_fidelity_estimation(circuit,
```

Comment:
At least one of the tests should inspect and verify the second return value (the intermediate results).

Reply:
Done. I didn't test everything because some of the results are random, but I think the coverage maybe is sufficient since I hit every object?